dashboard/config/scripts_json/csp4-2024.script_json

```diff
@@ -10078,5 +10078,45 @@
         "script.name": "csp4-2024"
       }
     }
+  ],
+  "skills": [
```

Comment:
Is the plan to eventually seed skills separately or will we always have a copy of each skill in the script?

It seems like the optimal solution would be to seed skills separately and just seed levels_skills from the script.

Reply:
It also seems like these are never called? I may be wrong on that though.